scheduler/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM balenalib/raspberrypi3-alpine-python
+
+ENV INITSYSTEM on
+
+COPY scripts /usr/src/
+RUN chmod +x /usr/src/*.sh
+
+# RUN (crontab -l; echo "0 8 * * * /usr/src/backlight_on.sh") | crontab -
+# RUN (crontab -l; echo "19 11 * * * /usr/src/backlight_off.sh") | crontab -
+
+ENTRYPOINT crond -f
```

Comment:
switch to using `CMD` because the balenalib base images use ENTRYPOINT to run `/usr/src/entry.sh` that sets up a few useful things for the container.